db/migrate/20240409222653_backfill_projects_latest_version_id.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class BackfillProjectsLatestVersionId < ActiveRecord::Migration[7.0]
+  def up
+    # This migration has already been run manually in production.
+    return if Rails.env.production?
+
+    Project
+      .in_batches(of: 10_000) do |projects|
+        sql = <<~SQL
+          WITH rows_to_update AS (
+            SELECT p.id as project_id, latest_version.id as latest_version_id
+            FROM projects p
+            LEFT JOIN LATERAL (
```

Comment:
🎱 Another way to do this would be to use a `rank() over partition by` clause to get the latest version per project. 

I think in either case it would be good to have a spec to test out this functionality to make sure it's not like setting all 10000 projects' latest_version_id to the one same version row.